packages/site/src/layout/SearchBox.tsx

```diff
@@ -33,8 +33,12 @@ export const SearchBox = ({
   const [search, setSearch] = useState("");
 
   const filteredDesignList = useMemo(() => {
-    return designList.filter(d =>
-      d.title.toLowerCase().includes(search.toLowerCase()),
+    return designList.filter(
+      d =>
+        d.title.toLowerCase().includes(search.toLowerCase()) ||
+        d.additionalMatches?.find(e =>
+          e.toLowerCase().includes(search.toLowerCase()),
+        ),
```

Comment:
@scotttjob this is literally a copy+paste of the `filteredComponentsList` function below, and we'll presumably want this for Hooks and whatnot as well? So _I'm_ fine with this for now but might be worth having one of yourself/Taylor/Becca/Aiden/Jacob take a peek at a "whatever you pass into search is case insensitive and accounts for additionalMatches" sort of standardized approach, because QAing the design changes it took me way too long to realize that `designList` wasn't hooked up the same way!

Reply:
Yup we can definitely make this better. I think it was components to start with and then it was cheap at the time to just duplicate a second for design. 

Now we're entering the territory where DRY-ing out the solution has more upside than a simple copy and paste for the third time on hooks.

I'll make sure to add a ticket for this.